src/modules/chats/components/chatlist/dolldata.js

```diff
@@ -0,0 +1,31 @@
+export const dollData = {
+    chatlist:[
+    {"id":"id0000001",
+    "name":"Паша",
+    "avatar":"",
+    "unread":5,
+    "time":"10:40",
+    "lastmessege": "Ха, прикольно"
+  },
+  {id: "id0000002",
+    "name":"Илья",
+    "avatar":"",
+    "unread":0,
+    "time":"10:40",
+    "lastmessege": "Вы: Не могу найти информацию"
+  },
+  { id:"id0000003",
+    "name":"Рабочая группа",
+    "avatar":"",
+    "unread":12,
+    "time":"Ср",
+    "lastmessege": "Изображение"
+  },
+  {id:"id0000004",
+    "name":"Квартиросъемщик",
+    "avatar":"",
+    "unread":0,
+    "time":"Пн",
+    "lastmessege": "Шумят очень сильно"
+  }
+]}
```

Comment:
От форматирования, достаточно сильно зависит, наглядность вашего кода

```suggestion
export const dollData = {
    chatlist: [
        {
            'id': 'id0000001',
            'name': 'Паша',
            'avatar': '',
            'unread': 5,
            'time': '10:40',
            'lastmessege': 'Ха, прикольно',
        },
        {
            id: 'id0000002',
            'name': 'Илья',
            'avatar': '',
            'unread': 0,
            'time': '10:40',
            'lastmessege': 'Вы: Не могу найти информацию',
        },
        {
            id: 'id0000003',
            'name': 'Рабочая группа',
            'avatar': '',
            'unread': 12,
            'time': 'Ср',
            'lastmessege': 'Изображение',
        },
        {
            id: 'id0000004',
            'name': 'Квартиросъемщик',
            'avatar': '',
            'unread': 0,
            'time': 'Пн',
            'lastmessege': 'Шумят очень сильно',
        },
    ],
};

```